charts/prometheus/values.yaml

```diff
@@ -892,12 +892,12 @@ server:
   ##
   readinessProbeInitialDelay: 30
   readinessProbePeriodSeconds: 5
-  readinessProbeTimeout: 30
+  readinessProbeTimeout: 4
   readinessProbeFailureThreshold: 3
   readinessProbeSuccessThreshold: 1
   livenessProbeInitialDelay: 30
   livenessProbePeriodSeconds: 15
-  livenessProbeTimeout: 30
+  livenessProbeTimeout: 4
```

Comment:
as livenessProbePeriodSeconds is 15 here i would got to 10 seconds for the livenessProbeTimeout.

Reply:
@monotek One thing I am considering is that surely the timeout threshold should remain the same for both probes? Given that the internal Prometheus implementation for both is very similar and both respond very quickly, it seems that 10 seconds is almost certainly overkill

Reply:
The liveness probe would restart the pod when its failing so imho is reasonable to give it some more time.

Reply:
That seems like a reasonable justification. I've incorporated your suggest change.